git/cmd.py

```diff
@@ -11,12 +11,13 @@
 import logging
 import os
 import signal
-from subprocess import Popen, PIPE, DEVNULL
+from subprocess import Popen, PIPE, DEVNULL, run, CalledProcessError
 import subprocess
 import threading
 from textwrap import dedent
+from pathlib import Path
 
-from git.compat import defenc, force_bytes, safe_decode
+from git.compat import defenc, force_bytes, safe_decode, is_win
```

Comment:
I think `is_win` [is deprecated](https://github.com/gitpython-developers/GitPython/blob/32c02d1c016b99dce9edc8895749fa936db85fab/git/compat.py#L31).